recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -322,12 +322,10 @@ FILES_libvulkan-imx-dev = "${includedir}/vulkan ${libdir}/libvulkan_VSI${SOLIBSD
 
 FILES_libopenvx-imx = " \
         ${libdir}/libOpenVX*${SOLIBS} \
-        ${libdir}/libOpenVXC${SOLIBS} \
-        ${libdir}/libOpenVXU${SOLIBS} \
         ${libdir}/libOvx*${SOLIBS} \
         ${libdir}/libovx*${SOLIBS} \
         "
-FILES_libopenvx-imx-dev = "${includedir}/VX ${libdir}/libopenVX${SOLIBSDEV} "
```

Comment:
The `libopenVX.so` should go to `-dev`. Is not provided?

Reply:
The libraries are not versioned. See line 106: `SOLIBS = "${SOLIBSDEV}`.

Reply:
This is conceptually wrong. It should be. 

This may introduce serious bugs on running systems if a new version adds or change symbols without a `soname` change.

Reply:
Thanks, Otavio. I have entered a bug internally for this.

Reply:
Check internally which soname it is intended and do as we did for the other libs, so this is not hold by their handling of it.

Reply:
In fact, changelog mentions it: 1.2